### PR TITLE
Updated Upstream (BungeeCord)

### PR DESCRIPTION
**Pull request by linsaftw**

### CODE REVIEW

File: BungeeCord-Patches/0030-Fix-potion-race-condition-on-Forge-1.8.9.patch [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+index 4b52c8b1..3a199578 100644
 --- a/protocol/src/main/java/net/md_5/bungee/protocol/AbstractPacketHandler.java
 +++ b/protocol/src/main/java/net/md_5/bungee/protocol/AbstractPacketHandler.java
-@@ -243,4 +243,14 @@ public abstract class AbstractPacketHandler
-     public void handle(FinishConfiguration finishConfiguration) throws Exception
+@@ -263,4 +263,14 @@ public abstract class AbstractPacketHandler
+     public void handle(CookieResponse cookieResponse) throws Exception
      {
      }
-+
```
Comment (Janmm14) [inline BungeeCord-Patches/0030-Fix-potion-race-condition-on-Forge-1.8.9.patch:15]: unneccessary change

File: BungeeCord-Patches/0041-Forge-is-a-first-class-citizen.patch [hunk 2]
```diff
@@ -18,18 +18,19 @@ index 066e740d..f25ff75a 100644
...
 --- a/proxy/src/main/java/net/md_5/bungee/conf/Configuration.java
 +++ b/proxy/src/main/java/net/md_5/bungee/conf/Configuration.java
-@@ -73,7 +73,7 @@ public abstract class Configuration implements ProxyConfig
+@@ -73,8 +73,8 @@ public abstract class Configuration implements ProxyConfig
      private Favicon favicon;
      private int compressionThreshold = 256;
      private boolean preventProxyConnections;
 -    private boolean forgeSupport;
+     private boolean rejectTransfers;
 +    private boolean forgeSupport = true; // Waterfall: default to enabled
```
Comment (Janmm14) [inline BungeeCord-Patches/0041-Forge-is-a-first-class-citizen.patch:30]: why change the order

File: BungeeCord-Patches/0048-Speed-up-some-common-exceptions.patch [hunk 3]
```diff
@@ -172,19 +172,19 @@ index 237955ab..d0bd4d75 100644
...
+ import lombok.AllArgsConstructor;
+ import lombok.Data;
+ import lombok.EqualsAndHashCode;
 +import javax.crypto.spec.SecretKeySpec;
 +
  import lombok.Getter;
  import lombok.RequiredArgsConstructor;
- import net.md_5.bungee.BungeeCord;
-@@ -470,6 +472,14 @@ public class InitialHandler extends PacketHandler implements PendingConnection
+ import lombok.ToString;
```
Comment (Janmm14) [inline BungeeCord-Patches/0048-Speed-up-some-common-exceptions.patch:186]: undo the import changes

File: BungeeCord-Patches/0048-Speed-up-some-common-exceptions.patch [hunk 4]
```diff
@@ -172,19 +172,19 @@ index 237955ab..d0bd4d75 100644
...
 +++ b/proxy/src/main/java/net/md_5/bungee/connection/InitialHandler.java
-@@ -15,6 +15,8 @@ import java.util.Set;
- import java.util.UUID;
- import java.util.logging.Level;
- import javax.crypto.SecretKey;
+@@ -21,6 +21,8 @@ import javax.crypto.SecretKey;
+ import lombok.AllArgsConstructor;
+ import lombok.Data;
+ import lombok.EqualsAndHashCode;
 +import javax.crypto.spec.SecretKeySpec;
```
Comment (Janmm14) [inline BungeeCord-Patches/0048-Speed-up-some-common-exceptions.patch:182]: This import change seems to be misplaced. Previously it was fitting in alphabetical order.

**Comment by electronicboy** [comment]
I generally do not care about trivial formatting stuff for this project anymore, once again, the upstream changelog is missing, however

**Review by electronicboy — APPROVED** [review]

**Pull request merged**